src/Cinelovers/Views/Movies/MovieDetailsView.xaml

```diff
@@ -0,0 +1,33 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<rxui:ReactiveContentPage x:Class="Cinelovers.Views.Movies.MovieDetailsView"
+                          xmlns="http://xamarin.com/schemas/2014/forms"
+                          xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                          xmlns:ff="clr-namespace:FFImageLoading.Forms;assembly=FFImageLoading.Forms"
+                          xmlns:rxui="clr-namespace:ReactiveUI.XamForms;assembly=ReactiveUI.XamForms"
+                          xmlns:vm="clr-namespace:Cinelovers.ViewModels.Movies;assembly=Cinelovers.ViewModels"
+                          x:TypeArguments="vm:MovieDetailsViewModel">
+    <ScrollView>
+        <StackLayout BackgroundColor="{StaticResource BackgroundColor}">
```

Comment:
Interesting way to do a platform independent border for the "card"!
There is one bug when the content is smaller than the screen: the end is the color of the border and not white as it should be.

![screenshot_1512773578](https://user-images.githubusercontent.com/21045769/33788527-04a7bace-dc5a-11e7-8ef5-a8a5d1919f29.png)

I think the list cell might have the same issue, but it is not visible there.

One other small detail is that you could probably just use a `ContentView`, on the top layouts since they have a single child.

Reply:
I couldn't see this happening, but I will try that again and fix what is needed here.